app/components/forms/events/view/create-access-code.js

```diff
@@ -0,0 +1,166 @@
+import Ember from 'ember';
+import FormMixin from 'open-event-frontend/mixins/form';
+import $ from 'jquery';
```

Comment:
Don't import this from `jquery`. It is already available as a part of ember at `Ember.$`. Use that please.